config/.env.local

```diff
@@ -1,6 +1,6 @@
 # IMPORTANT: do not put anything sensitive in this file
 
-PUBLIC_AQUIFER_API_KEY=""
+PUBLIC_AQUIFER_API_KEY="8b55d5dbb4dd4cd5a7d1e85d6d6d738b"
```

Comment:
Note that this key is [already in source control](https://github.com/BiblioNexusStudio/content-manager-web/blob/e7d01d199894fddd27f59b56797536604e5e9ffe/config/.env.dev-local#L3).